staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -95,3 +110,24 @@ func (a *AdmissionOptions) Validate() []error {
 	errs := []error{}
 	return errs
 }
+
+// enabledPluginNames makes use of RecommendedPluginOrder and DefaultOffPlugins fields
+// to prepare a list of plugin names that are enabled.
+func (a *AdmissionOptions) enabledPluginNames() []string {
+	onlyEnabledPluginNames := []string{}
+	for _, pluginName := range a.RecommendedPluginOrder {
+		disablePlugin := false
+		for _, disabledPluginName := range a.DefaultOffPlugins {
```

Comment:
add a // TODO indicating that a later flag for "user specified off" will also be handled here, but won't mutate `DefaultOffPlugins`?